core-bundle/tests/Picker/TablePickerProviderTest.php

```diff
@@ -332,7 +332,7 @@ public function testGetUrlWithPtableAndMultipleTables(): void
         $provider = $this->createTableProvider(
             $this->mockFrameworkWithDcaLoader('tl_article'),
             $this->mockRouterWithExpectedParams($params),
-            $this->mockConnectionForQuery('tl_article', 15, ['pid' => 1])
```

Comment:
if this argument is not passed, it's not very obvious/not tested that the `mockPickerConfig` and `mockConnectionForQuery` expect the same ID. Removing the argument would not make code more obvious, even though I agree the argument might be kinda unnecessary. But maybe we should use a class constant instead?

Reply:
Yes, we could do this, however, that is not what this PR is about.

Reply:
I have reverted the changes in 3fa8d4e0e56b089d8476fab4976d0d13729d7f2d and used arbitrary IDs instead of `15` everywhere.